config/src/projects/deversifi.ts

```diff
@@ -44,6 +44,7 @@ export const deversifi: Project = {
         'LRC',
         'OMG',
         'BAT',
+        'DVF',
```

Comment:
Please also add the `associatedToken: 'DVF',` property to the project definition. Place it between `bridges` and `details`